app/assets/stylesheets/extend/bootstrap.scss

```diff
@@ -58,6 +58,19 @@
   float: none;
 }
 
+// link protocol button
+.btn-info {
+  color: #fff !important;
+  background-color: $color-theme-primary !important;
+  border-color: #2385ba !important;
+}
+
+.btn-info:hover {
+  color: #fff !important;
+  background-color: #2793ce !important;
+  border-color: #2384b8 !important;
+}
+
 @media (max-width: 1000px) {
   .navbar-header {
     float: none;
```

Comment:
Are those color changes in `bootstrap.scss` really needed?
1.) We never use color constants directly in `.scss` files, but always define them elsewhere.
2.) We should define our own style in `scinote.scss`, not this file.